Earthfile

```diff
@@ -1,1085 +0,0 @@
-ARG GO_VERSION=1.23
-# renovate: datasource=docker depName=hadolint/hadolint
-ARG HADOLINT_VERSION=2.12.0-alpine
-# renovate: datasource=docker depName=renovate/renovate
-ARG RENOVATE_VERSION=39
-# renovate: datasource=docker depName=koalaman/shellcheck-alpine versioning=semver
-ARG SHELLCHECK_VERSION=v0.10.0
-
-ARG IMAGE
-ARG IMAGE_REPOSITORY_ORG=quay.io/kairos
-ARG AURORABOOT_IMAGE=quay.io/kairos/auroraboot:$AURORABOOT_VERSION
-ARG K3S_VERSION
-ARG CGO_ENABLED=0
-
-all:
-  ARG SECURITY_SCANS=true
-
-  ARG TARGETARCH
-  ARG --required FAMILY # The dockerfile to use
-  ARG --required FLAVOR # The distribution E.g. "ubuntu"
-  ARG --required FLAVOR_RELEASE # The distribution release/version E.g. "20.04"
-  ARG --required VARIANT
-  ARG --required MODEL
-  ARG --required BASE_IMAGE # BASE_IMAGE is the image to apply the strategy (aka FLAVOR) on. E.g. ubuntu:20.04
-
-
-  BUILD +base-image
-  IF [ "$SECURITY_SCANS" = "true" ]
-      BUILD +image-sbom
-      BUILD +trivy-scan
-      BUILD +grype-scan
-  END
-  BUILD +iso
-  BUILD +netboot
-  BUILD +ipxe-iso
-
-# For PR building, only image and iso are needed
-ci:
-  ARG SECURITY_SCANS=true
-
-  # args for base-image target
-  ARG --required FLAVOR
-  ARG --required FLAVOR_RELEASE
-  ARG --required BASE_IMAGE
-  ARG --required MODEL
-  ARG --required VARIANT
-  ARG --required FAMILY
-
-
-  BUILD +base-image
-  IF [ "$SECURITY_SCANS" = "true" ]
-    BUILD +image-sbom
-    BUILD +trivy-scan
-    BUILD +grype-scan
-  END
-  BUILD +iso
-
-all-arm:
-  ARG --required FLAVOR
-  ARG --required FLAVOR_RELEASE
-  ARG --required BASE_IMAGE
-  ARG --required MODEL
-  ARG --required VARIANT
-  ARG --required FAMILY
-
-  ARG COMPRESS_IMG=true
-  ARG SECURITY_SCANS=true
-
-  BUILD --platform=linux/arm64 +base-image
-  IF [ "$SECURITY_SCANS" = "true" ]
-      BUILD --platform=linux/arm64 +image-sbom
-      BUILD --platform=linux/arm64 +trivy-scan
-      BUILD --platform=linux/arm64 +grype-scan
-  END
-
-  IF [ "$MODEL" = "nvidia-jetson-agx-orin" ]
-    BUILD +prepare-nvidia-l4t
-  ELSE
-    BUILD +arm-image
-  END
-
-arm-container-image:
-  ARG --required FLAVOR
-  ARG --required FLAVOR_RELEASE
-  ARG --required BASE_IMAGE
-  ARG --required VARIANT
-  ARG --required FAMILY
-  ARG --required MODEL
-  BUILD --platform=linux/arm64 +base-image
-
-all-arm-generic:
-  ARG --required FLAVOR
-  ARG --required FLAVOR_RELEASE
-  ARG --required BASE_IMAGE
-  ARG --required VARIANT
-  ARG --required FAMILY
-  BUILD --platform=linux/arm64 +iso --MODEL=generic
-
-build-and-push-golang-testing:
-    ARG GO_VERSION
-    FROM golang:$GO_VERSION
-    # Enable backports repo for debian for swtpm
-    RUN . /etc/os-release && echo "deb http://deb.debian.org/debian $VERSION_CODENAME-backports main contrib non-free" > /etc/apt/sources.list.d/backports.list
-    RUN apt update
-    RUN apt install -y qemu-system-x86 qemu-utils git swtpm && apt clean
-    SAVE IMAGE --push $IMAGE_REPOSITORY_ORG/golang-testing:${GO_VERSION}
```

Comment:
Please remind me where was this used?

Reply:
it did create the same image that it itself consumed in a different target no? @mauromorales ?